### DISCUSSION TITLE
Welcome to FLwatch Discussions!

### DISCUSSION THREAD

**Discussion by poml88** [poml88/FLwatch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jens24** [comment]
It would be great if BG# could be a CarPlay widget!! I would like to see my BG#s on my dashboard in the car. This would really help in safely driving.

**Reply by poml88** [reply]
Hi, it already exists. You can mirror the LiveActivity and / or put a widget on CarPlay.

**Comment by cknoop** [comment]
Hey, I’ve recently started using the CAM APS FX app. The sensor is started directly within that app, which is presumably why no readings are coming through via LibreLink Up. I’m getting an "Invalid Activation Response"... Can FLwatch still be used in this setup, or am I out of luck? Thanks.

**Reply by poml88** [reply]
I don’t think so. A) the Libre can do only one connection to one device and B) you would need to know your receiver ID.

**Reply by cknoop** [reply]
no chance to get the reciever id from anywhere? Naja, thx for reply! ;-)

**Reply by poml88** [reply]
If CamAPS is using the „official“ receiverID we can retrieve it. Just try in FLwatch.
